src/components/bulkConcepts/component/ConceptTable.jsx

```diff
@@ -16,7 +16,7 @@ const ConceptTable = ({
     );
   }
   const filter = { filterMethod: filterConcept, filterAll: true };
-    return (
+  return (
```

Comment:
I think such formatting should be handled on another ticket to remove ESlint warnings.

Reply:
@Tittoh I would think for a change like this where I haven't necessarily written new code is fine?

Reply:
@Tittoh I just think that's an Linter issue... otherwise we are going to be pushing code with warnings from the Linter... and overtime they will accumulate...